### PR TITLE
Diverts post command to Yogpost instead of headcoder things

### PR DESCRIPTION
**Pull request by swissloaf**
adds yogpost

🆑 
rscadd: Yogpost role for post
/🆑

### CODE REVIEW

**Comment by swissloaf** [comment]
me and nich are best friends

**Comment by swissloaf** [comment]
YogPost role is real now

File: models/Discord/Commands/DiscordCommandPost [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+  constructor(subsystem) {
+    super("post", "post the input as yogbot.", 'post', subsystem);
+  }
+
+  onRun(message, permissions, args) {
+    var config = this.subsystem.manager.getSubsystem("Config").config;
+
+    if(args.length < 1) {
+        message.delete(0);
```
Comment (AshCorr) [inline models/Discord/Commands/DiscordCommandPost]: Change this to `args.length > 0` and change the delete and return for `message.channel.send(args.join(" "))`
Little bit less code this way.

Comment (swissloaf) [reply models/Discord/Commands/DiscordCommandPost]: @AshCorr what do you want me to change the delete and return to

**Comment by nichlas0010** [comment]
Mane, adding roles for specific yogbot permissions is stupid when yogbot already has an integrated permissions system.

**Comment by swissloaf** [comment]
ASHCORR

**Pull request merged**